sparkmagic/sparkmagic/controllerwidget/manageendpointwidget.py

```diff
@@ -122,3 +127,12 @@ def get_info_endpoint_widget(self, endpoint, url):
             text = "No sessions on this endpoint."
 
         return self.ipywidget_factory.get_html(text, width=width)
+
+    def initialize_kerberos(self, endpoint):
```

Comment:
@aggFTW I think it's better not to destroy the kerberos ticket. There may be multiple processes using the kerberos ticket. Consider livy and jupyter running on the same server, destroying the ticket will effect livy.